tests/view_tests/tests/test_debug.py

```diff
@@ -731,6 +732,31 @@ def test_reporting_of_nested_exceptions(self):
         self.assertIn(implicit_exc.format("<p>Second exception</p>"), text)
         self.assertEqual(3, text.count("<p>Final exception</p>"))
 
+    def test_highlight_error_position(self):
+        request = self.rf.get("/test_view/")
+        try:
+            try:
+                raise AttributeError(mark_safe("<p>Top level</p>"))
+            except AttributeError as explicit:
+                try:
+                    raise ValueError(mark_safe("<p>Second exception</p>")) from explicit
+                except ValueError:
+                    raise IndexError(mark_safe("<p>Final exception</p>"))
+        except Exception:
+            # Custom exception handler, just pass it into ExceptionReporter
+            exc_type, exc_value, tb = sys.exc_info()
+
+        reporter = ExceptionReporter(request, exc_type, exc_value, tb)
+        html = reporter.get_traceback_html().replace(" ", "")
+        if PY311 and not sys._xoptions.get("no_debug_ranges", False):
```

Comment:
Move this condition to a `unittest.skipIf` decorator on the test, because the other branch isn’t testing anything new on Python <3.11?

Reply:
According to PEP-657, positions can also be deactivated using the `PYTHONNODEBUGRANGES` environment variable, so we should check both.